src/Symfony/Bundle/TwigBundle/Tests/DependencyInjection/TwigExtensionTest.php

```diff
@@ -193,11 +193,45 @@ public function testTwigLoaderPaths($format)
             array('namespaced_path1', 'namespace1'),
             array('namespaced_path2', 'namespace2'),
             array('namespaced_path3', 'namespace3'),
-            array(__DIR__.'/Fixtures/Resources/TwigBundle/views', 'Twig'),
             array(__DIR__.'/Fixtures/templates/bundles/TwigBundle', 'Twig'),
             array(realpath(__DIR__.'/../..').'/Resources/views', 'Twig'),
             array(realpath(__DIR__.'/../..').'/Resources/views', '!Twig'),
-            array(__DIR__.'/Fixtures/Resources/views'),
+            array(__DIR__.'/Fixtures/templates'),
+        ), $paths);
+    }
+
+    /**
+     * @group legacy
+     * @dataProvider getFormats
+     * @expectedDeprecation Templates directory "%s" is deprecated since Symfony 4.2, use "%s" instead.
```

Comment:
can we include some vital parts of the paths? something like `"%sResource/view%s" ... "%stemplates%s"` could work?

Reply:
Much better now :) thanks!